src/storage/txn/actions/flashback_to_version.rs

```diff
@@ -191,7 +197,10 @@ pub mod tests {
         let cm = ConcurrencyManager::new(TimeStamp::zero());
         let mut txn = MvccTxn::new(start_ts, cm);
         let snapshot = engine.snapshot(Default::default()).unwrap();
-        let mut reader = SnapshotReader::new_with_ctx(version, snapshot, &ctx);
+        let mut reader = ReaderWithStats::new(
+            SnapshotReader::new_with_ctx(version, snapshot, &ctx),
+            &mut statistics,
```

Comment:
Do you want to assert about whether there is something in `statistics` as you're using it in tests?

Reply:
Not necessarily needed, I removed it first.